uint8 mode_req
---
uint8 mode_resp
